### ISSUE TITLE
warning for writing intermediate raw files

### ISSUE THREAD

**Issue by F-said** [NDCLab/baseEEG#218]
**Describe the bug**
During the writing of all intermediate raw files using, the following error pops up:
> This filename (...) does not conform to MNE naming conventions. All raw files should end with raw.fif, raw_sss.fif, raw_tsss.fif, raw.fif.gz, raw_sss.fif.gz, raw_tsss.fif.gz or _meg.fif

**Expected behavior**
No light warnings stating violation of MNE standards should appear.

**Screenshots**

**Comment by georgebuzzell** [comment]
@F-said this occurs when writing the raw data after each preprocessing step? what is the format of the data when written after each preprocessing step? Isn't it the bids/brain vision format and not fif?

@DMRoberts

**Comment by F-said** [comment]
@georgebuzzell the raw object itself is in bids/brain vision format, but when saved to a file the type isn't specified. It looks like the MNE package wants us to specify it to be `fif`. Would that be acceptable?

**Comment by F-said** [comment]
per meeting #220 temporarily fif file

**Comment by F-said** [comment]
Now that the bug has been named it causes an exception that stops the entire pipeline 🤦

```
OSError: The filename (...\preprocessed\sub-NDARAB793GL3\ses-01\eeg\sub-NDARAB793GL3_ses-01_task-Video1_run-01_eeg_filter_data) for file type raw must end with .fif or .fif.gz. 
```

Naming all intermediate files `fif` until better extension is found.

**Comment by F-said** [comment]
closed with #230
